modulefiles/RDAS/wcoss2.intel.lua

```diff
@@ -0,0 +1,102 @@
+help([[
+Load environment for running the GDAS application with Intel compilers and MPI.
+]])
+
+local pkgName    = myModuleName()
+local pkgVersion = myModuleVersion() or "1.0"
+local pkgNameVer = myModuleFullName()
+
+prepend_path("MODULEPATH", "/apps/ops/test/spack-stack-1.6.0-nco/envs/nco-intel-19.1.3.304/install/modulefiles/Core")
+
+load("PrgEnv-intel/8.3.3")
+load("stack-intel/19.1.3.304")
+load("stack-cray-mpich/8.1.9")
+load("stack-python/3.10.13")
+load("craype/2.7.17")
+load("cray-pals/1.3.2")
+load("cmake/3.23.1")
+load("gettext/0.19.7")
+load("pcre2/10.42")
+load("curl/8.4.0")
+load("zlib/1.2.13")
+load("git/2.35.3")
+load("git-lfs/2.11.0")
+load("pkg-config/0.29.2")
+load("hdf5/1.14.0")
+load("parallel-netcdf/1.12.2")
+load("netcdf-c/4.9.2")
+load("nccmp/1.9.0.1")
+load("netcdf-fortran/4.6.1")
+load("nco/5.0.6")
+load("parallelio/2.5.10")
+load("boost/1.83.0")
+load("bufr/12.0.1")
+load("ecbuild/3.7.2")
+load("openjpeg/2.5.0")
+load("eigen/3.4.0")
+load("openblas/0.3.24")
+load("eckit/1.24.5")
+load("fftw/3.3.10")
+load("fckit/0.11.0")
+load("fms/2023.04")
+load("esmf/8.5.0")
+load("atlas/0.35.0")
+load("sp/2.5.0")
+load("gsl-lite/0.37.0")
+load("libjpeg/2.1.0")
+load("libpng/1.6.37")
+load("libxt/1.1.5")
+load("libxmu/1.1.4")
+load("libxpm/3.5.12")
+load("libxaw/1.0.13")
+load("udunits/2.2.28")
+load("ncview/2.1.9")
+load("netcdf-cxx4/4.3.1")
+load("core/rocoto/1.3.5")
+load("prod_util/2.0.14")
+
+load("py-setuptools/63.4.3")
+load("py-jinja2/3.1.2")
+load("py-netcdf4/1.5.8")
+load("py-pybind11/2.11.1")
+load("py-pycodestyle/2.11.0")
+load("py-pyyaml/5.4.1")
+load("py-scipy/1.10.1")
+load("py-xarray/2023.7.0")
+load("py-f90nml/1.4.3")
+load("py-pip/23.1.2")
+load("py-bottleneck/1.3.7")
+load("py-numexpr/2.8.4")
+load("py-six/1.16.0")
+load("py-python-dateutil/2.8.2")
+load("py-pytz/2023.3")
+load("py-pandas/1.5.3")
+load("py-numpy/1.24.4")
+load("py-markupsafe/2.1.3")
+load("py-cftime/1.0.3.4")
+load("py-packaging/23.1")
+
+setenv("CC","cc")
+setenv("CXX","CC")
+setenv("FC","ftn")
+
+-- stack-intel/19.1.3.304 sets compiler flags "-static-libgcc -static-libstdc++ -Bstatic -lstdc++ -Bdynamic -lm -lpthread"
+-- flags trigger "warning #10315: specifying -lm before files may supersede the Intel(R) math library and affect performance"
+-- TODO: remove unsetenv lines when complier flags are removed from the wcoss2 spack-stack build
+unsetenv("CFLAGS")
+unsetenv("CXXFLAGS")
+unsetenv("FFLAGS")
+
+local mpiexec = '/opt/cray/pals/1.3.2/bin/mpirun'
+local mpinproc = '-n'
+setenv('MPIEXEC_EXEC', mpiexec)
+setenv('MPIEXEC_NPROC', mpinproc)
+
+setenv("CRTM_FIX","/lfs/h2/emc/da/noscrub/emc.da/GDASApp/fix/crtm/2.4.0")
+setenv("GDASAPP_TESTDATA","/lfs/h2/emc/da/noscrub/emc.da/GDASApp/testdata")
+setenv("GDASAPP_UNIT_TEST_DATA_PATH", "/lfs/h2/emc/da/noscrub/emc.da/GDASApp/unittestdata")
```

Comment:
We can safely remove line 95-97?

Reply:
@guoqing-noaa  So, CRTM_FIX is not used too?

Reply:
We get that from `${HOMErrfs}/fix`

Reply:
it looks like GDASApp already uses crtm/2.4.0?

Do wen want to update crtm as well?  @xyzemc @HaidaoLin-NOAA

Reply:
@TingLei-NOAA I think we don't need line 95.  @xyzemc may test that on wcoss2.

You can see we don't have `CRTM_FIX` in the module files in other machines.
If it is not needed, we better to remove it to avoid confusions.

Reply:
Done @guoqing-noaa

Reply:
Thanks, @TingLei-NOAA